ui/src/wishDetails/WishDetails.js

```diff
@@ -1,30 +1,102 @@
 import React, { Component } from 'react';
+import WishDetailsService from '../services/WishDetailsService'
+import Rocket from '../assets/icn_To_Go_Rocket_White_Inside_130x130.png'
+import Alien from '../assets/icn_To_Meet_Alien_White_Inside_130x130.png'
+import Astronaut from '../assets/icn_To_Be_Astronaut_White_Inside_130x130.png'
+import Telescope from '../assets/icn_To_See_Telescope_White_Inside_130x130.png'
 
 export default class WishDetails extends Component {
   constructor(props) {
     super();
     this.state = {
-      id: 0
+      id: '',
+      wishDetails: {
+        id: '',
+        child: {
+          firstName: '',
+          lastName: ' ',
```

Comment:
is the space here desired?

Reply:
Fixed this in latest commit

Reply:
Also changed to name instead of firstName, lastName